plugin/helpers_bots.go

```diff
@@ -138,6 +148,10 @@ func (p *HelpersImpl) ShouldProcessMessage(post *model.Post, options ...ShouldPr
 		return false, nil
 	}
 
+	if !messageProcessOptions.AllowWebhook && post.Props["from_webhook"] == true {
```

Comment:
shouldn't this be `post.Props["from_webhook"] == "true"` instead?

Reply:
or should it be both? because it's a `interface{}` oh boy....

Reply:
Good catch. It should be `post.Props["from_webhook"] == "true"`.